src/components/Loader/Loader.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+export const Loader: React.FC = () => {
+  return (
```

Comment:
```suggestion
export const Loader: React.FC = () => (
```

can be a bit simplify